### PR TITLE
Issue #57

### LINKED ISSUE #57: Update REAME and documentation

### PR DESCRIPTION
**Pull request by xUnholy**
Fixed #57 

Fixed:
- Up & Down command descriptions
- Updated golangci-lint to version 1.16.+

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,54 +3,195 @@
...
 
-The namespace can be overriden via the `--namespace` option, for example: `kube-compose --namespace ci up`.¯
+### Build And Package
 
-# Advanced usage
-If you require that an application is not started until one of its dependencies is healthy, you can add `condition: service_healthy` to the `depends_on`, and give the dependency a [Docker healthchecks](https://docs.docker.com/engine/reference/builder#healthcheck).
+You can compile the kube-compose binary using either Go or Docker-compose.
 
-Docker healthchecks are converted into [Readiness Probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).
+Using Go:
```
Comment (jbrekelmans) [inline README.md]: `-o kube-compose` not needed, mind updating .travis file aswell?

File: README.md [hunk 2]
```diff
@@ -3,54 +3,195 @@
...
+
+Using Makefile (**Recommended**):
+
+```make
+make releases
 ```
+
+*Note: Will build for Linux, MacOS (darwin), and Windows.*
+
+Using Docker-compose:
```
Comment (jbrekelmans) [inline README.md]: Should remove docker-compose.yml for building and Dockerfile

File: README.md [hunk 3]
```diff
@@ -3,54 +3,195 @@
...
-# Testing
-Use `kubectl` or `oc` to set the target Kubernetes namespace and the service account of kube-compose.
+### Testing
+
+Use `kubectl` to set the target Kubernetes namespace and the service account of kube-compose.
 
 Run `kube-compose` with the test [docker-compose.yml](test/docker-compose.yml):
-```
+
+```bash
```
Comment (jbrekelmans) [inline README.md]: /kube-compose

File: .golangci.yml [hunk 4]
```diff
@@ -62,5 +62,4 @@ issues:
 # golangci.com configuration
 # https://github.com/golangci/golangci/wiki/Configuration
 service:
-  golangci-lint-version: 1.15.x # use the fixed version to not introduce new linters unexpectedly
-  prepare:
\ No newline at end of file
+  golangci-lint-version: 1.16.x # use the fixed version to not introduce new linters unexpectedly
```
Comment (russ-c) [inline .golangci.yml]: this comment is confusing - maybe "specific version"

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+GOMOD=$(GOCMD) mod
+BINARY_NAME=kube-compose
+SYSTEMS=darwin linux windows
+
+all: build test
+
+build: 
+	$(GOBUILD) -o $(BINARY_NAME) -v
+
+tests:
```
Comment (russ-c) [inline Makefile]: test

File: cmd/down.go [hunk 6]
```diff
@@ -9,7 +9,7 @@ import (
 
 var downCmd = &cobra.Command{
 	Use:   "down",
-	Short: "A brief description of your command",
+	Short: "Stop and remove containers, networks, images, and volumes running on K8s",
```
Comment (jbrekelmans) [inline cmd/down.go]: `Deletes the pods of the specified docker compose services. If all docker compose services would be deleted then the Kubernetes services are also deleted.`

File: README.md [hunk 7]
```diff
@@ -3,54 +3,195 @@
...
+
+Start individual services defined in docker-compose.yml
+
+```bash
+kube-compose up service-1
+
+kube-compose up service-1 service-2
+```
+
+### Detach mode
```
Comment (russ-c) [inline README.md]: Detached

**Pull request merged**